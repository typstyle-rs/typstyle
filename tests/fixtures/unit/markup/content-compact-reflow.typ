/// typstyle: wrap_text
/// typstyle: include=content-compact.typ
