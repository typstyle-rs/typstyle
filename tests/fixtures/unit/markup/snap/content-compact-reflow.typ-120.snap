---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/markup/content-compact-reflow.typ
---
/// typstyle: wrap_text
/// typstyle: include=content-compact.typ
#[- 111]
#[- 222]
#[- 333]
#[/ 444:]
#[/ 555:]
#[/ 666:]
#[$12345$]
#[= 123]

#[- aaa bbb]
#[
  / ccc:
    ddd
]

#[*strong
*]
#[_
  emph_]
#[*
  _
    strong-emph
  _
*]

#[#{}]
#[#[]]
#[ #{} ]
#[ #[ ] ]
#[#{
  //
}]
#[#[
  //
]]
#[#[#[#[
  aaa
]]]]

#[
  /*  */
  000]
#[
  /*  */
  \#
]
#[
  /*  */
  \\
]
#[
  //
  --
]

#[$ 111111111111111111111111111111111111 $]
#[$
  2222222222222222222222222222222222222
$]

#[
  $ a $
  \
]
#[
  $ b $
  \
  \
]

#[
  $ c $
  <l2>
]
#[
  $ d $ <l1>
  <l2>
]
